SU-HSDO/stanford_mrc
# Mrcd8 183 basic page ct

# READY FOR REVIEW

# Summary
- Adding a Basic page with a wysiwyg to MRC.

# Needed By (Date)
- end of sprint

# Urgency
- low

# Steps to Test

1. Provising from branch with same name in mrc_blt
2. Checkout this branch and enable Basic Page
3. Make sure it works as expected.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-183
- https://github.com/SU-HSDO/mrc_blt/pull/9
